openwisp_radius/base/models.py

```diff
@@ -1046,6 +1057,13 @@ class Meta:
     def __str__(self):
         return self.organization.name
 
+    @property
+    def freeradius_allowed_hosts_list(self):
+        addresses = []
+        if self.freeradius_allowed_hosts:
+            addresses = self.freeradius_allowed_hosts.split(',')
+        return addresses
+
```

Comment:
I have not tested this code, so please make necessary changes where needed. :sweat_smile:

Reply:
`freeradius_allowed_hosts_list(self)` is actually only used to convert string to list. I am not sure how making changes in it can reflect on the `admin` change form.
Although, I think I'll read this response again in the morning before giving a proper response.